test/hack/resource/clean/main.go

```diff
@@ -31,19 +31,22 @@ import (
 	"github.com/aws/karpenter/test/hack/resource/pkg/resourcetypes"
 )
 
-const expirationTTL = time.Hour * 12
 const sweeperCleanedResourcesTableName = "sweeperCleanedResources"
 
 func main() {
-	var clusterName string
-	if len(os.Args) == 2 {
-		clusterName = os.Args[1]
-	}
+	expiration := flag.String("expiration", "12h", "define the expirationTTL of the resources")
```

Comment:
```suggestion
	expiration := flag.String("--expiration", "12h", "define the expirationTTL of the resources")
```

More standard syntax